core/samples/net/page.md

```diff
@@ -0,0 +1,6 @@
+# Networking {#examples-net-networking}
+
+@brief Using the @ref core-net module.
+
+- @subpage examples-core-net-udp-sockets - @copybrief examples-core-net-udp-sockets
+- @subpage examples-core-net-tcp-sockets - @copybrief examples-core-net-tcp-sockets
```

Comment:
In the TCP sample `page.md` you used `examples-core-tcp-sockets`, make sure to change it to `examples-core-net-tcp-sockets` otherwise the link won't work.